.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 ui/build
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode
```

Comment:
nit: all of these should be in a [global git ignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)

Reply:
Thanks! I never included any gitignore in my global config file, I'll leave this here, as we were also ignoring other IDEs. But my global gitignore is already working! 👍